### PR TITLE
Feat RSPY510: Update CADIP mappings

### PR DESCRIPTION
**Pull request by Padeanu**

### CODE REVIEW

**Review by jgaucher-cs — COMMENTED** [review]
@Padeanu how can we test your fixes ? You did not implement new tests, did you test it by hand ? @vprivat-ads should we test this with real stations ?

**Comment by Padeanu** [comment]
> @Padeanu how can we test your fixes ? You did not implement new tests, did you test it by hand ? @vprivat-ads should we test this with real stations ?

Launch Stac Browser and observe

**Comment by vprivat-ads** [comment]
@jgaucher-cs  We are going to test them with real stations (with the fix for rspy-515 as well)

**Comment by jgaucher-cs** [comment]
> > @Padeanu how can we test your fixes ? You did not implement new tests, did you test it by hand ? @vprivat-ads should we test this with real stations ?
> 
> Launch Stac Browser and observe

OK so I let Vincent and you test it with real stations, otherwise the PR is OK for me.

**Review by jgaucher-cs — APPROVED** [review]

File: services/cadip/rs_server_cadip/cadip_utils.py [hunk 1]
```diff
@@ -195,6 +196,11 @@ def link_assets_to_session(session_data, assets_dict, mapper):
             }
             asset: Asset = Asset(title=asset_dict.pop("id"), roles=["cadu"], **asset_dict)
             feature.assets.update({asset.title: asset})
+        end_date = min(
```
Comment (vprivat-ads) [inline services/cadip/rs_server_cadip/cadip_utils.py]: @Padeanu it should be max, not min ?

**Review by vprivat-ads — APPROVED** [review]

**Pull request merged**